sorc/build_gfs_bufrsnd.sh

```diff
@@ -21,6 +21,11 @@ if [ ! -d "../exec" ]; then
   mkdir ../exec
 fi
 
+if [ $target = "wcoss2" ]; then
+  export SIGIO_LIB4=$SIGIO_LIB
```

Comment:
Aren't these usually set by the sigio module?

Reply:
Needed to support both stack (WCOSS2) and pre-stack (elsewhere).